package.json

```diff
@@ -37,6 +41,15 @@
     "eslint-plugin-eslint-comments": "^3.2.0",
     "eslint-plugin-functional": "^3.0.2",
     "ts-node": "^9.0.0",
+    "typescript": "^4.4.3",
+    "cpx": "^1.5.0"
+  },
+  "workspaces": [
+    "packages/*",
+    "providers/*"
+  ]
+}
+
```

Comment:
Looks like the merge had an issue, you need to remove these lines so we can have a valid package.json json format.